README.md

```diff
@@ -81,6 +81,10 @@ The following input parameters are supported for the Endor Labs GitHub action:
 | `scan_secrets` | Scan source code repository and generate findings for secrets. See also `scan_git_logs`. (Default: `false`) |
 | `scan_summary_output_type` | Set the desired output format to `table`, `json`, `yaml`, or `summary`. (Default: `json`) |
 | `tags` | Specify a list of user-defined tags to add to this scan. Tags can be used to search and filter scans later. |
+| `use-bazel` | Enable the usage of Bazel for the scan. (Default: `false`)|
+| `bazel_exclude_targets` | The list of Bazel targets that are not to be scanned. |
```

Comment:
How about calling these options `bazel_targets_include` and `bazel_targets_exclude` to match `bazel_targets_query`?

Reply:
Actually this is actually on purpose;
- include and exclude ends with targets since we expect a list of targets.
- query is at the end because we expect a query.

My goal was that following the semantic of the flags, users could expect what input it needs.